bpf/ipsec.h

```diff
@@ -166,6 +167,7 @@ int BPF_KRETPROBE(xfrm_input_kretprobe) {
     return update_flow_with_ipsec_return(xfrm_ret, INGRESS);
 }
 
+// https://elixir.bootlin.com/linux/v6.14.8/source/net/xfrm/xfrm_output.c#L743
```

Comment:
Those hooks supported on older kernel too not specific to v6.14.8 so not sure why added this comment

Reply:
I just find it helpful to get a direct link to the code for kprobes -  don't mean to pin a version in particular, any version would work